docs/notification_system.md

```diff
@@ -43,179 +48,188 @@ The currently defined Notification Types are:
 * Webhook
 * Grafana
 
-Each of these have their own configuration and behavioral semantics and testing them may need to be approached in different ways.   The following sections will give as much detail as possible.
+Each of these have their own configuration and behavioral semantics and testing them may need to be approached in different ways. The following sections will give as much detail as possible.
+
 
 ## Email
 
-The email notification type supports a wide variety of smtp servers and has support for ssl/tls connections and timeouts.
+The email notification type supports a wide variety of SMTP servers and has support for SSL/TLS connections and timeouts.
 
-### Testing considerations
+### Testing Considerations
 
 The following should be performed for good acceptance:
 
-* Test plain authentication
-* Test SSL and TLS authentication
-* Verify single and multiple recipients
+* Test plain authentication.
+* Test SSL and TLS authentication.
+* Verify single and multiple recipients.
 * Verify message subject and contents are formatted sanely.  They should be plaintext but readable.
 
 ### Test Service
 
-Either setup a local smtp mail service here are some options:
+Set up a local SMTP mail service.  Some options are listed below:
+
+* Postfix service on galaxy: https://galaxy.ansible.com/debops/postfix/
+* Mailtrap has a good free plan that should provide all of the features necessary: https://mailtrap.io/
```

Comment:
A docker container is another good option here:

`docker run --network="tools_default" -p 25:25 -e maildomain=mail.example.com -e smtp_user=user:pwd --name postfix -d catatnight/postfix`